.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E722
-max-line-length = 160
+ignore = E722,W503
```

Comment:
FYI https://www.flake8rules.com/rules/W503.html

I approve of ignoring this rule

Reply:
Do you prefer W504 or W503?

Reply:
I tend towards W504, if someone wanted to roll it out that would be interesting to see.